docs/blog/moving-portland-prague-2020-online.rst

```diff
@@ -1,59 +1,52 @@
 .. post:: Apr 29, 2020
    :tags: portland-2020, prague-2020, conferences, covid-19
 
-Moving our Portland and Prague 2020 conferences online
-======================================================
+Moving our 2020 conferences online
```

Comment:
We haven't confirmed AU yet.

Reply:
I know, but it makes the title more digestable. :-p We could explicitly mention it in the body?